INSTALL.rst

```diff
@@ -244,42 +237,9 @@ venv
   # (On Windows, please use ``skimage-dev\Scripts\activate``)
   source ~/envs/skimage-dev/bin/activate
   # Install main development and runtime dependencies
-  pip install -r requirements.txt
-  # Install build dependencies of scikit-image
-  pip install -r requirements/build.txt
-  # Build scikit-image from source
-  spin build
-  # The new version lives under `${PWD}/build-install/.../site-packages`.
-  # Test your installation
-  spin test
-  # Build docs
-  spin docs
-  # Try the new version in IPython
-  spin ipython
-
-conda
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-When using conda for development, we
-recommend adding the conda-forge channel for the most up-to-date version
-of many dependencies.
-Some dependencies we use (for testing and documentation) are not available
-from the default Anaconda channel. Please follow the official
-`conda-forge installation instructions <https://conda-forge.org/#about>`_
-before you get started.
-
-.. code-block:: sh
-
-  # Create a conda environment named ``skimage-dev``
-  conda create --name skimage-dev
-  # Activate it
-  conda activate skimage-dev
-  # Install main development and runtime dependencies
-  conda install -c conda-forge --file requirements/default.txt
-  conda install -c conda-forge --file requirements/test.txt
```

Comment:
I understand that these two files are difficult to keep in sync. with an additional conda environment file it doesn't get easier.... i know...

But scipy has this file,
https://github.com/scipy/scipy/blob/main/environment.yml#L8

and it might be appropriate to just keep it a `default + testing` dependencies for simplicity